### PR TITLE
chore(deps): update schema-utils to v2

### PR DESCRIPTION
**Pull request by hiroppy**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?

updated

### Motivation / Use-Case

closes: #2129

### Breaking Changes

no

### CODE REVIEW

File: test/options.test.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 const { join } = require('path');
-const ValidationError = require('schema-utils/src/ValidationError');
+const {
+  default: ValidationError,
+} = require('schema-utils/dist/ValidationError');
```
Comment (evilebottnawi) [inline test/options.test.js]: no need do this, name can be changed and location too, we should use try/catch and use snapshots to validate

**Comment by hiroppy** [comment]
Updated

**Comment by hiroppy** [comment]
Thank you for pointing it. I fixed

**Comment by evilebottnawi** [comment]
Need update snapshots and can be merged when CI green

**Comment by evilebottnawi** [comment]
Done in next

**Pull request closed without merging**